<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XMMEMainWindow</class>
 <widget class="QMainWindow" name="XMMEMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>138</y>
    <width>584</width>
    <height>323</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>AgentGUI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../xmme.qrc">
    <normaloff>:/icons/xmme/images/euracelogo.png</normaloff>:/icons/xmme/images/euracelogo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>X Machines Model Hierarchy</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <widget class="XModelView" name="modelView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="editing">
       <attribute name="title">
        <string>Edit Properties</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="XMM">
       <attribute name="title">
        <string>XMML Information</string>
       </attribute>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QTextBrowser" name="informationView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>584</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionQuoit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionQuoit">
   <property name="icon">
    <iconset resource="../xmme.qrc">
     <normaloff>:/icons/xmme/images/application-exit.png</normaloff>:/icons/xmme/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../xmme.qrc">
     <normaloff>:/icons/xmme/images/help-about.png</normaloff>:/icons/xmme/images/help-about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../xmme.qrc">
     <normaloff>:/icons/xmme/images/document-save.png</normaloff>:/icons/xmme/images/document-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save &amp;As</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../xmme.qrc">
     <normaloff>:/icons/xmme/images/document-open.png</normaloff>:/icons/xmme/images/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../xmme.qrc">
     <normaloff>:/icons/xmme/images/document-new.png</normaloff>:/icons/xmme/images/document-new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XModelView</class>
   <extends>QTreeView</extends>
   <header>../src/XModelView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../xmme.qrc"/>
 </resources>
 <connections/>
</ui>
